### DISCUSSION TITLE
aust人才济济啊

### DISCUSSION THREAD

**Discussion by Restayu** [nyakang/AutoLogin#5] [General]
学长牛逼

**Comment by nyakang** [comment]
> 学长牛逼

😆😆😆
好久没管这个项目了，破解限速貌似不咋行了。

**Reply by Restayu** [reply]
大佬 我好像出了点问题  能加一下我QQ吗 2430385640

**Reply by polargiraffe** [reply]
没弄成功，返回值是0，没登上去
破解限速原理是啥？
学校现在教育网关了

**Reply by nyakang** [reply]
> 没弄成功，返回值是0，没登上去
> 破解限速原理是啥？
> 学校现在教育网关了

本人现在不在学校，无法测试了。破解限速应该就是简单的“多播”，发多次请求。

**Reply by nyakang** [reply]
> 亲手试验改成sleep(0.01)，即刻突破千兆，原代码是sleep(3) 总之多谢楼主了，爱你！

enjoy it! 
**别滥用哦**

**Reply by polargiraffe** [reply]
我跑pt下载电影用，上传速度没提升，流控应该抓不到我，没有到处宣扬传播，真转播了也没几个人感兴趣QAQ，不过之前在贴吧看见过有人分享exe文件的登录器，应该和这个一个原理,后面登录器成功概率太低了，贴子也删了~

**Reply by Restayu** [reply]
调试一晚上了，还是无法百分之百成功。俺一定要再完善一下破解系统（但愿

**Reply by nyakang** [reply]
> 调试一晚上了，还是无法百分之百成功。俺一定要再完善一下破解系统（但愿

小伙子很有干劲啊💎
